packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSlider.tsx

```diff
@@ -1,23 +1,58 @@
-import React, { useCallback, useState, ChangeEventHandler } from 'react';
+import React, { useCallback, ChangeEventHandler, useMemo } from 'react';
 import styled from 'styled-components';
 import { AnimatePresence, HTMLMotionProps, motion } from 'framer-motion';
-import { Range, Warning, useTheme, motionEasing } from '@trezor/components';
+import { Range, Warning, useTheme, motionEasing, Icon, variables } from '@trezor/components';
 import { Translation } from '@suite-components';
-import { useSelector, useActions, useAnonymityStatus } from '@suite-hooks';
+import { useAnonymityStatus } from '@suite-hooks';
 import { AnonymityStatus } from '@suite-constants/coinjoin';
-import { selectSelectedAccount } from '@wallet-reducers/selectedAccountReducer';
-import * as coinjoinActions from '@wallet-actions/coinjoinAccountActions';
 
-const Container = styled.div`
+const Container = styled.div<{ isBlurred: boolean }>`
     display: flex;
     flex-direction: column;
+    position: relative;
+
+    > div {
+        transition: filter 0.15s;
+        filter: ${({ isBlurred }) => isBlurred && 'blur(7px)'};
+    }
 `;
 
 const Slider = styled(Range)`
     margin-top: 0;
     cursor: pointer;
 `;
 
+const Label = styled.span`
+    display: flex;
+    align-items: center;
+
+    > :first-child {
+        margin-right: 3px;
+    }
+`;
+
+const DisabledMessage = styled.p`
+    position: absolute;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    color: ${({ theme }) => theme.TYPE_DARK_GREY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    backdrop-filter: grayscale(0.5);
+    filter: ${({ theme }) => `drop-shadow(0px 0px 8px ${theme.BG_LIGHT_GREY})`};
+
+    > :first-child {
+        margin-right: 6px;
+    }
+`;
+
+const RedText = styled.span`
+    margin-right: 2px;
+    color: ${({ theme }) => theme.TYPE_RED};
+`;
+
 const expandAnimation: HTMLMotionProps<'div'> = {
     initial: { height: 0, marginTop: 0, opacity: 0 },
```

Comment:
nit: consider to add marginBottom
<img width="669" alt="image" src="https://user-images.githubusercontent.com/3729633/208458349-5b28b013-5b84-465b-a5e8-c1d7b7f27c97.png">